### PR TITLE
Add github CI action

### PR DESCRIPTION
**Pull request by jasonleenaylor**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2)

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run eslint:github-action
```
Comment (imnasnainaec) [inline .github/workflows/node.js.yml]: Do we need to add a script for `eslint:github-action` to `package.json`?

Comment (jasonleenaylor) [reply .github/workflows/node.js.yml]: We need to add something. Just trying to work out what the simplest something is. :)

**Review by imnasnainaec — APPROVED** [review]
> Reviewed 1 of 4 files at r3.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[.github/workflows/node.js.yml, line 28 at r3](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2#-MHwVVs3DUNMbDZ3aOXU:-MHwVVs45bwDQhibRPiT:buy7x8p) ([raw file](https://github.com/sillsdev/react-localize-crowdin/blob/c55ca55498c9eeb075b690270f602182c13d26c3/.github/workflows/node.js.yml#L28)):*
> ```YAML
>         node-version: ${{ matrix.node-version }}
>     - run: npm ci
>     - run: npm run build --if-present
> ```

Do we expect to have a `build` script (soon)?

**Review by jasonleenaylor — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[.github/workflows/node.js.yml, line 28 at r3](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2#-MHwVVs3DUNMbDZ3aOXU:-MHwWXrI8HX8U58NzSdr:b-vnqdyx) ([raw file](https://github.com/sillsdev/react-localize-crowdin/blob/c55ca55498c9eeb075b690270f602182c13d26c3/.github/workflows/node.js.yml#L28)):*
<details><summary><i>Previously, imnasnainaec (D. Ror.) wrote…</i></summary><blockquote>

Do we expect to have a `build` script (soon)?
</blockquote></details>

Ah, good eye. :smile: Is there something we can replace that line with that will work as things are right now?

**Review by imnasnainaec — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[.github/workflows/node.js.yml, line 28 at r3](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2#-MHwVVs3DUNMbDZ3aOXU:-MHwa8IxB0GrWW1YZwrJ:b58ushv) ([raw file](https://github.com/sillsdev/react-localize-crowdin/blob/c55ca55498c9eeb075b690270f602182c13d26c3/.github/workflows/node.js.yml#L28)):*
<details><summary><i>Previously, jasonleenaylor (Jason Naylor) wrote…</i></summary><blockquote>

Ah, good eye. :smile: Is there something we can replace that line with that will work as things are right now?
</blockquote></details>

I figure we delete that line if it doesn't have any near future purpose. Does it need a replacement?

**Review by imnasnainaec — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[.github/workflows/node.js.yml, line 28 at r3](https://reviewable.io/reviews/sillsdev/react-localize-crowdin/2#-MHwVVs3DUNMbDZ3aOXU:-MHwb9_I-M4kW5dqi1kL:b-7xiu1) ([raw file](https://github.com/sillsdev/react-localize-crowdin/blob/c55ca55498c9eeb075b690270f602182c13d26c3/.github/workflows/node.js.yml#L28)):*
<details><summary><i>Previously, imnasnainaec (D. Ror.) wrote…</i></summary><blockquote>

I figure we delete that line if it doesn't have any near future purpose. Does it need a replacement?
</blockquote></details>

Perhaps it's a standard element: https://docs.github.com/en/actions/guides/building-and-testing-nodejs

**Pull request merged**